### ISSUE TITLE
Frame-Selector >>> does not work with `element=xxx` selector

### ISSUE THREAD

**Issue by Snooz82** [MarketSquare/robotframework-browser#2662]
when first getting the frames from a page and the using them, it does not work.

```robotframework
Search Text
    ${frames}    Get Elements     frame, iframe
    FOR    ${f}    IN    ${frames}
        ${count}    Get Element Count    ${f} >>> "search text"
        IF    $count    RETURN
    END
    Fail    Element not found
```

**Comment by Snooz82** [comment]
A somewhat hacky workaround for now is:

```robotframework
Search Text
    ${amount}    Get Element Count    frame, iframe
    FOR    ${i}    IN RANGE    ${amount}
        ${count}    Get Element Count    frame, iframe >> nth=${i} >>> "search text"
        IF    $count    RETURN
    END
    Fail    Element not found
```

**Comment by Snooz82** [comment]
Should get fixed by #2911

**Comment by Snooz82** [comment]
Can confirm the fix with #2917
